app/os_windows.go

```diff
@@ -59,7 +61,10 @@ type window struct {
 	invMu sync.Mutex
 }
 
-const _WM_WAKEUP = windows.WM_USER + iota
+const (
+	_WM_WAKEUP = windows.WM_USER + iota
+	_WM_OPEN_URL
```

Comment:
The `_WM_OPEN_URL` name and value is misleading because it's not a window message identifier like `WM_PAINT` or our `_WM_WAKEUP`. It should probably be something like

`const copyDataURLType = 0xffffff00`

with a sufficiently large constant that minimizes the risk of collision.